### PR TITLE
Fixed notification shimmer

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
Fixed notification shimmer

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 

As I have checked and found that it is working fine for LTR and for RTL it is still having issue. Please check the video for reference.

**VIdeo 1 - LTR**

https://github.com/bagisto/bagisto/assets/92721837/43317769-31c4-4c15-b3c3-b6d7e10950da

**VIdeo 1 - RTL**


https://github.com/bagisto/bagisto/assets/92721837/5658095a-c342-43cd-8376-ba30e85c2649

**Comment by ashishkumar-webkul** [comment]
Checked and found that the issue has been fixed and working fine for the shimmer effect in RTL and LTR view. Please check the video for reference.

**Video**


https://github.com/bagisto/bagisto/assets/92721837/1b5c1807-a038-43ef-96d8-55d80a08342e

File: packages/Webkul/Admin/src/Resources/views/components/shimmer/notifications/index.blade.php [hunk 1]
```diff
@@ -2,38 +2,38 @@
...
                     </div>    
                 @endfor
             </div>
 
             <!-- Notifications List -->
-            <div class="journal-scroll grid max-h-[calc(100vh-330px)] overflow-auto">
-                @for ($i = 1; $i < 7; $i++)
-                    <div class="flex h-14 items-start gap-1.5 p-4 hover:bg-gray-50 dark:hover:bg-gray-950">
+            <div class="grid max-h-[calc(100vh-330px)]">
+                @for ($i = 1; $i < 6; $i++)
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/components/shimmer/notifications/index.blade.php:29]: Here, revert the removed classes and update the foreach loop to 7 lengths for the big screen.

**Review by amit-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/views/components/shimmer/notifications/index.blade.php (hunk 1 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Resources/views/components/shimmer/notifications/index.blade.php]: There is no fixed ratio for a big screen, so there is no need to fix the length for static shimmer. If we use the scroll class, then the ratio will be greater than 10.

**Pull request merged**